che-theia-plugins.yaml

```diff
@@ -58,14 +58,14 @@ plugins:
     extensions:
       - https://github.com/VSCodeVim/Vim/releases/download/v1.16.0/vim-1.16.0.vsix
   - repository:
-      url: https://github.com/testthedocs/vscode-vale
-      revision: 0.9.1
+      url: https://github.com/errata-ai/vale-vscode/
+      revision: 0.12.0
```

Comment:
A few things:
* The latest version is 0.14.2, not 0.12.0: https://github.com/errata-ai/vale-vscode/releases
* The revision field must match a valid tag/branch/SHA1 from the repository, which in this case would be `v0.12.0`, or `v0.14.2` if we go with 0.14.2.

Reply:
Why would that matter? The vsix is from open-vsx, not built.

Reply:
Also: the latest version published to open-vsx is 0.12. Since we can't use the marketplace 🤷

Reply:
why not use https://github.com/errata-ai/vale-vscode/releases/download/v0.14.2/vale-server-0.14.2.vsix ?

Reply:
> The revision field must match a valid tag/branch/SHA1 from the repository, which in this case would be v0.12.0, or v0.14.2 if we go with 0.14.2.

@ericwill the CI job is not checking that revision is valid ?

@tsmaeder AFAIK this field is used by the plug-in 'automatic-check' that generates report https://eclipse.github.io/che-plugin-registry/

Reply:
> > The revision field must match a valid tag/branch/SHA1 from the repository, which in this case would be v0.12.0, or v0.14.2 if we go with 0.14.2.
> 
> @ericwill the CI job is not checking that revision is valid ?

It was for the old `vscode-extensions.json` file, it has not yet been ported to the new `che-theia-plugins.yaml` system. It's on the radar for next sprint: https://github.com/eclipse/che/issues/18729